### ISSUE TITLE
Document reference architecture for project

### ISSUE THREAD

**Issue by yelodevopsi** [varianter/vibes#45]
Reference architecture for backend and frontend should be documented in Markdown-files and committed to the project.
```[tasklist]
### Tasks
- [ ] https://github.com/varianter/vibes/issues/72
- [ ] https://github.com/varianter/vibes/issues/73
```

**Comment by yelodevopsi** [comment]
@jonasbjoralt Should document the PR #69 with this task maybe?

**Comment by yelodevopsi** [comment]
Task partially solved with restructuring in #69 

Documentation remains

**Comment by yelodevopsi** [comment]
Closing, since archtest is implemented, and no further docs are required.
